src/tests/tests.test.js

```diff
@@ -0,0 +1,101 @@
+import {
+    getSameItemsInCart,
+    getRangeLimit,
+    getSortedList,
+    getParams,
+    updateCartSum,
+    getTotalStock,
+    showRangedProducts,
+    getSummaryTotalAmount,
+    getSummaryTotalPriceBeforePromo,
+    getSummaryTotalPriceAfterPromo,
+} from './basicFunctions';
+import { SEARCHED_ITEM_IN_CART, ITEMS, SORTED_ITEMS, RANGED_ITEMS } from './dataForTests';
+
+describe('Given the item', () => {
+    test('Then I expect the total amount of the item in cart', () => {
```

Comment:
In this case been better if u try to write description with "should" construction.
For example: 
```
  ✓ should contain a button
  ✓ should dispatch an event on click
  ✓ should show the name of the user
  ✓ should show the type of widget
```